### ISSUE TITLE
Why I only get mIoU=67.63 on Potsdam,I set the batch size=2 and train model on one gpu due to the resouce limit

### ISSUE THREAD

**Issue by kouyuanbo** [lyhnsn/GLOTS#6]
This is my result,Is it just because the batch size is too small？:
```
2024-11-19 04:56:27,417 - mmseg - INFO - 
+--------------------+-------+-------+
|       Class        |  IoU  |  Acc  |
+--------------------+-------+-------+
| impervious_surface | 78.74 | 89.68 |
|      building      | 84.24 | 91.55 |
|   low_vegetation   | 69.55 | 85.65 |
|        tree        | 65.69 | 76.69 |
|        car         | 80.45 | 92.89 |
|      clutter       | 27.12 | 32.74 |
+--------------------+-------+-------+
2024-11-19 04:56:27,417 - mmseg - INFO - Summary:
2024-11-19 04:56:27,417 - mmseg - INFO - 
+-------+-------+------+
|  aAcc |  mIoU | mAcc |
+-------+-------+------+
| 84.69 | 67.63 | 78.2 |
+-------+-------+------+
2024-11-19 04:56:27,417 - mmseg - INFO - Exp name: swin_beit-base_640x640_160k_potsdam.py
2024-11-19 04:56:27,417 - mmseg - INFO - Iter(val) [2016]	aAcc: 0.8469, mIoU: 0.6763, mAcc: 0.7820, IoU.impervious_surface: 0.7874, IoU.building: 0.8424, IoU.low_vegetation: 0.6955, IoU.tree: 0.6569, IoU.car: 0.8045, IoU.clutter: 0.2712, Acc.impervious_surface: 0.8968, Acc.building: 0.9155, Acc.low_vegetation: 0.8565, Acc.tree: 0.7669, Acc.car: 0.9289, Acc.clutter: 0.3274
```

this is my config:
```
_base_ = [
    '../_base_/models/swin_beit.py', '../_base_/datasets/potsdam_640.py',
    '../_base_/default_runtime_vis.py', '../_base_/schedules/schedule_160k.py'
]

# model cfg
model = dict(
    pretrained='pretrain/beitv2_base_patch16_224_pt1k_ft21k.pth',
    decode_head=dict(num_classes=6), # reversely index
    auxiliary_head=dict(num_classes=6),
    test_cfg=dict(crop_size=(640, 640)))

# optimizer cfg
optimizer = dict(
    _delete_=True,
    type='AdamW',
    lr=3e-5,
    betas=(0.9, 0.999),
    weight_decay=0.05,
    constructor='LayerDecayOptimizerConstructor',
    paramwise_cfg=dict(num_layers=12, layer_decay_rate=0.9))

lr_config = dict(
    _delete_=True,
    policy='poly',
    warmup='linear',
    warmup_iters=1500,
    warmup_ratio=1e-6,
    power=1.0,
    min_lr=0.0,
    by_epoch=False)


data = dict(
    # val=dict(pipeline=test_pipeline),
    # test=dict(pipeline=test_pipeline),
    samples_per_gpu=2)
```

**Comment by lyhnsn** [comment]
Yes. Batch size is an important factor that affects the normalization of the training data, especially when the scale is large. Please try a larger batch size.

**Comment by kouyuanbo** [comment]
> Yes. Batch size is an important factor that affects the normalization of the training data, especially when the scale is large. Please try a larger batch size.

Ok，thanks a lot for your reply!
